pkg/globalconfig/global_config.go

```diff
@@ -70,6 +66,34 @@ type GlobalConfig struct {
 	ProjectList                      map[string]*ProjectInfo `yaml:"project_info"`
 }
 
+// New() returns a default GlobalConfig
+func New() GlobalConfig {
+
+	cfg := GlobalConfig{
+		RequiredDockerComposeVersion: RequiredDockerComposeVersionDefault,
+		InternetDetectionTimeout:     nodeps.InternetDetectionTimeoutDefault,
+		TableStyle:                   "default",
+		RouterHTTPPort:               "80",
+		RouterHTTPSPort:              "443",
+		LastStartedVersion:           "v0.0",
+		NoBindMounts:                 nodeps.NoBindMountsDefault,
+		UseTraefik:                   nodeps.UseTraefikDefault,
```

Comment:
I see that these should be constants, I don't want to do it right now but will try to fix that up in the traefik PR.